File: cognizent_INT/Basic.cs
using System;
public class Basic
{
      public static void Main(String[] args)
      {
            // int n=11;
            Console.WriteLine("Enter the number : ");
            int n=int.Parse(Console.ReadLine());
            Console.WriteLine(n%2==0?"The number is even":n+" this is a odd number");
      }
}

File: cognizent_INT/Basic.java
import java.util.Scanner;

public class Basic {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the number : ");
        int n = Integer.parseInt(scanner.nextLine());
        System.out.println(n % 2 == 0 ? "The number is even" : n + " this is a odd number");
    }
}
